templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -35,16 +35,38 @@
           },
         ]"
       >
-        <ul class="page-menu-item fa-ul">
-          <page-menu-item
-            v-for="row in rows"
-            :key="row.node.id"
-            :node="row.node"
-            :lockRows="lockRows"
-            :disabled="disabledRow(row.node)"
-            @scroll-to="scrollToRef"
-          />
-        </ul>
+        <div v-for="(menu, index) in menuGroups" :key="index" class="menu-wrapper">
+          <b-card class="menu">
+            <div>
+              <b-card-text v-if="index === 0">
+                {{ getMenuName(index) }}
+              </b-card-text>
+              <b-card-text v-else v-b-toggle="`collapse-${index}`">
+                {{ getMenuName(index) }}
+              </b-card-text>
+            </div>

```

Comment:
Could this be replaced with something like:
```suggestion
            <b-card-text v-b-toggle="index === 0 ? `` : `collapse-${index}`">
              {{ getMenuName(index) }}
            </b-card-text>
```